### ISSUE TITLE
Calendar appointment in HTML

### ISSUE THREAD

**Issue by Biocodr** [Microsoft/UWPQuickStart#1]
Hi there

I saw that you are using AppointmentDetailsKind.Html in the EventDetails.xaml.cs class. I tried to achieve the same but the HTML is not rendered and the calendar just shows all the HTML tags. Does it work for you? Is there something else I need to declare?

Best regards

**Comment by unniravindranathan** [comment]
@Biocodr Yes, it does work for the code in this repo. Can you share a repro of what you are trying? Also, what is the e-mail client that you have associated by default?
